azure-nextgen-ts-functions-raw/index.ts

```diff
@@ -0,0 +1,250 @@
+// Copyright 2016-2021, Pulumi Corporation.  All rights reserved.
+
+import * as resources from "@pulumi/azure-nextgen/resources/latest";
+import * as storage from "@pulumi/azure-nextgen/storage/latest";
+import * as web from "@pulumi/azure-nextgen/web/latest";
+import * as pulumi from "@pulumi/pulumi";
+
+// Create a resource group for Windows App Service Plan
+const resourceGroup = new resources.ResourceGroup("windowsrg", {
+    resourceGroupName: "windowsrg",
+    location: "westus",
+});
+
+const storageAccount = new storage.StorageAccount("functionsa2021", {
+    accountName: "functionsa2021",
+    resourceGroupName: resourceGroup.name,
+    kind: storage.Kind.StorageV2,
+    sku: {
+        name: storage.SkuName.Standard_LRS,
+    },
+});
+
+const plan = new web.AppServicePlan("windows-plan", {
+    name: "functions-windows-plan",
+    resourceGroupName: resourceGroup.name,
+    sku: {
+        name: "Y1",
+        tier: "Dynamic",
+    },
+});
+
+const container = new storage.BlobContainer("container", {
+    containerName: "blob-container",
+    accountName: storageAccount.name,
+    resourceGroupName: resourceGroup.name,
+    publicAccess: storage.PublicAccess.None,
+});
+
+const dotnetBlob = new storage.Blob("dotnetBlob", {
+    blobName: "dotnetBlob",
+    resourceGroupName: resourceGroup.name,
+    accountName: storageAccount.name,
+    containerName: container.name,
+    type: storage.BlobType.Block,
+    source: new pulumi.asset.FileArchive("./dotnet/bin/Debug/netcoreapp3.1/publish"),
+});
+
+const dotnetBlobSignedURL = signedBlobReadUrl(dotnetBlob, container, storageAccount, resourceGroup);
+
+const dotnetApp = new web.WebApp("http-dotnet", {
+    name: "httpdotnet",
+    resourceGroupName: resourceGroup.name,
+    serverFarmId: plan.id,
+    kind: "FunctionApp",
+    siteConfig: {
+        appSettings: [
+            { name: "runtime", value: "dotnet" },
+            { name: "FUNCTIONS_WORKER_RUNTIME", value: "dotnet" },
+            { name: "WEBSITE_RUN_FROM_PACKAGE", value: dotnetBlobSignedURL },
+            { name: "WEBSITE_NODE_DEFAULT_VERSION", value: "~12" },
```

Comment:
Probably not needed for non-node functions?

Reply:
Yup - removed.